diff -ru leechcraft-0.6.60/src/interfaces/itexteditor.h leechcraft-0.6.60.b/src/interfaces/itexteditor.h
--- leechcraft-0.6.60/src/interfaces/itexteditor.h	2013-12-30 10:12:25.000000000 +0200
+++ leechcraft-0.6.60.b/src/interfaces/itexteditor.h	2014-12-23 00:44:17.404761928 +0200
@@ -29,6 +29,7 @@
 
 #pragma once
 
+#include <functional>
 #include <QPair>
 #include <QList>
 #include <QRegExp>
diff -ru leechcraft-0.6.60/src/util/qml/plotitem.h leechcraft-0.6.60.b/src/util/qml/plotitem.h
--- leechcraft-0.6.60/src/util/qml/plotitem.h	2013-12-30 10:12:25.000000000 +0200
+++ leechcraft-0.6.60.b/src/util/qml/plotitem.h	2014-12-22 22:44:56.710547227 +0200
@@ -29,6 +29,7 @@
 
 #pragma once
 
+#include <functional>
 #include <QDeclarativeItem>
 #include <util/utilconfig.h>
 
